tests/storage/migration.go

```diff
@@ -269,10 +269,8 @@ var _ = SIGDescribe("[Serial]Volumes update with migration", Serial, func() {
 			Eventually(func() bool {
 				vm, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
```

Comment:
nit and not directly related: This Expect should be local to Eventually (passing `g Gomega` and using `g.Expect`)

Reply:
We should find an agreement on this :slightly_smiling_face: 
Generally speaking, you are right, but there was a discussion about this in the past, where the counterpart said that we don't want to retry in an eventually block if an API error occurs.